### ISSUE TITLE
Items in the open are not collected

### ISSUE THREAD

**Issue by Csimbi** [fireundubh/fallout4#34]
Some items laying out in the open are not collected.
I noticed this with Cap Stash (there is one in Concord, had to pick it up manually), weapons (again, a 10mm in Concord is laying on the street).
Are these not considered 'valuable items'?
Could you add armor and weapon classes separately and raise the limit to 6?
(What's the point of the limit anyway?)
Thanks!

**Comment by lordzues40** [comment]
there is a previous version that does not have the caps. it loots every single vanilla item.  it you go to the second to last post in the forum "Is it possible to simply loot everything from every "vanilla" source? and in rireundubh's reply click on the 4th step you can get the uncapped version. in this version I have noticed like cap stash hand placed weapons and armors it doesn't collect but In my opinion that is a very lil inconvenience.

**Comment by fireundubh** [comment]
Armor and weapons are randomly generated, so AFAIK, they cannot be auto looted like other items.

**Comment by Csimbi** [comment]
Yeah, I found that out last night when I looked under the hood:
Bethesda did not include a function to locate references of classes - only forms and keywords - so you are collecting items only on the form lists.
I saw that the Cap Stash activator is on the valuables form list - yet they don't get collected, even when they are in an arm's reach.
I don't think it's an ownership issue because the one in Concord is in the ruins (should not be owned by anyone).
